### ISSUE TITLE
Node modules not loading on Windows

### ISSUE THREAD

**Issue by StarLederer** [HermanLederer/mp3-tag-assistant#3]
This issue is probably caused by electron-builder not bundling things properly for windows because of pnpm, however there is a chance something got broken since 3.0.0-rc.1. Details will be added as the problem is researched.

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'axios'
Require stack:
at Module._resolveFilename (node:internal/modules/cjs/loader:934: 1 5)
at Function.n._resolveFilename
at Module._load (node:internal/modules/cjs/loader:779:27)
at Function.c._load
at Module.require (node:internal/modules/cjs/loader: lü:19)
at require 18)
at (C:\Program
Files\M etash ndclo u d. I .js:39: 1 5)
at Module._compile (node:internal/modules/cjs/loader:1110: 14)
at Object.Module._extensions.js (node:internal/modules/cjs/loader: 1140:10)
at Module.load
```

_Originally posted by @JVariance in https://github.com/StarLederer/metashine/issues/1#issuecomment-1280996766_

**Comment by StarLederer** [comment]
I got Metashine to (mostly) work in Wine (for some reason Spotify data is not loading). Don't want to rush to creating a new release so @JVariance, could you please follow build instructions in CONTRIBUTING.md and see if first, the instructions actually work, and second, if the issue at hand is resolved.

If the app works, please also check if you get any spotify results on the assistant tab, i really hope it not working is a Wine-specific error in which case *won't fix*. Otherwise will make a separate issue for that.

**Comment by JVariance** [comment]
Bad news 😕 I get the following error on startup:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'native-addon'
Require stack:
at Module._resolveFilename (node:internal/modules/cjs/loader:g34: 1 5)
at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1128)
at Module._load
at Functiomc._load (node:electron/js2c/asar_bundle:5:13331)
at Module.require (node:internal/modules/cjs/loader: lü:19)
at require (node:internal/modules/cjs/helpers:93: 18)
at Object.<anonymous>
.js: 1 10:27)
at Module._compile (node:internal/modules/cjs/loader:1110: 14)
at Object.Module._extensions.js (node:internal/modules/cjs/loader: 1140:10)
at Module.load
```

**Comment by JVariance** [comment]
Don't know if it's helpful, but I had to manually create the 'x86_64-pc-windows-gnu' folder inside 'target' and move the 'release' folder inside it which originally was created directly inside 'target'. (so the knownArtifactPaths in package.mjs should be updated).

**Comment by StarLederer** [comment]
Oops, that was an oversight. Fixed in [b9d12ed](https://github.com/StarLederer/metashine/commit/b9d12ed73dff34814d96429acc6ea016052a6b9e)

> so the knownArtifactPaths in package.mjs should be updated

Exactly that.

When you copied the artifact manually, did the build succeed and did the app work?

**Comment by JVariance** [comment]
No, unfortunately it didn't work either 😔

**Comment by StarLederer** [comment]
Turned out to be another oversight. At some point I somehow deleted / forgot to commit a package.json file in packages/native-addon. Fixed in [d724c16](https://github.com/StarLederer/metashine/commit/d724c160dca3e7932eb569ed3aea39020a57a73c) so closing the issue, however, feel free to comment to let me know if it also works for you now.

Regardless of that, **please don't use Metashine 3 yet** on important files. After additional testing with the newly successful build i found some bugs in the rust code (see #4)

**Comment by JVariance** [comment]
> Turned out to be another oversight. At some point I somehow deleted / forgot to commit a package.json file in packages/native-addon. Fixed in [d724c16](https://github.com/StarLederer/metashine/commit/d724c160dca3e7932eb569ed3aea39020a57a73c) so closing the issue, however, feel free to comment to let me know if it also works for you know.
> 
> Regardless of that, **please don't use Metashine 3 yet** on important files. After additional testing with the newly successful build i found some bugs in the rust code (see #4)

I can confirm that everything works as expected now! 👍🏼
